packages/manager/apps/web/client/app/hosting/hosting.html

```diff
@@ -215,6 +215,7 @@ <h1 class="d-inline-block mb-0 word-break">
         </wuc-autorenew-invite>
 
         <div data-ovh-alert="{{alerts.tabs}}"></div>
+        <div data-ovh-alert="{{alerts.database}}"></div>
```

Comment:
Using the `alerts.tab` doesn't suit the need to display alerts ?

Reply:
Actually no, because both could be displayed at the same time !